.eslintrc.json

```diff
@@ -13,10 +13,16 @@
   ],
   "parser": "@typescript-eslint/parser",
   "rules": {
+    "quotes": "off",
     "max-len": [
       2,
-      100
```

Comment:
bitte die max length auf einem sinnvollen wert lassen.
flutter hat hart in die sprache auf 80 kodiert.
80 finde ich bisschen hart. Bei 200 kann man auch keine grenze setzten.
Man muss sich einfach daran gewöhnen.
Mit 120 kann ich leben

Reply:
Das Problem waren die Kommentare. Mit den 100 im Code habe ich aber keine Probleme. Ich finde die Kommentare dürfen gerne länger sein.

Reply:
Seperater PR, oder willst du das gleich hier fixen?
```
"max-len": [
      "warn",
      {
        "code": 120,
        "ignoreComments": true
      }
    ],
```

Reply:
done